spec/tapioca/dsl/compilers/active_job_spec.rb

```diff
@@ -115,6 +115,36 @@ def perform_now(user_id); end
               RBI
               assert_equal(expected, rbi_for(:NotifyJob))
             end
+
+            it "generates correct RBI file for subclass with block argument" do
+              add_ruby_file("job.rb", <<~RUBY)
+                class NotifyJob < ActiveJob::Base
+                  def perform(user_id, &block)
```

Comment:
Can we add a case where the block parameter is NOT named `block` as well?

Reply:
I updated the current test with a block parameter not called `block`. I don't see why we need two tests for the same behaviour (checking if the param is a block param).